modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -87,20 +87,17 @@ enum ImwriteFlags {
 
 //! Imwrite PNG specific flags
 enum ImwritePNGFlags {
-       IMWRITE_PNG_STRATEGY_DEFAULT      = 0,
-       IMWRITE_PNG_STRATEGY_FILTERED     = 1,
-       IMWRITE_PNG_STRATEGY_HUFFMAN_ONLY = 2,
-       IMWRITE_PNG_STRATEGY_RLE          = 3,
-       IMWRITE_PNG_STRATEGY_FIXED        = 4
+       IMWRITE_PNG_STRATEGY_DEFAULT      = 0, //!< have no explanation yet
+       IMWRITE_PNG_STRATEGY_FILTERED     = 1, //!< have no explanation yet
+       IMWRITE_PNG_STRATEGY_HUFFMAN_ONLY = 2, //!< have no explanation yet
+       IMWRITE_PNG_STRATEGY_RLE          = 3, //!< have no explanation yet
+       IMWRITE_PNG_STRATEGY_FIXED        = 4  //!< have no explanation yet
```

Comment:
I think it is better to add some brief description here: these flags alter zlib behaviour, you can find documentation [here](http://www.zlib.net/manual.html#Advanced) --> function _deflateInit2_. It says:

>The strategy parameter is used to tune the compression algorithm. Use the value Z_DEFAULT_STRATEGY for normal data, Z_FILTERED for data produced by a filter (or predictor), Z_HUFFMAN_ONLY to force Huffman encoding only (no string match), or Z_RLE to limit match distances to one (run-length encoding). Filtered data consists mostly of small values with a somewhat random distribution. In this case, the compression algorithm is tuned to compress them better. The effect of Z_FILTERED is to force more Huffman coding and less string matching; it is somewhat intermediate between Z_DEFAULT_STRATEGY and Z_HUFFMAN_ONLY. Z_RLE is designed to be almost as fast as Z_HUFFMAN_ONLY, but give better compression for PNG image data. The strategy parameter only affects the compression ratio but not the correctness of the compressed output even if it is not set appropriately. Z_FIXED prevents the use of dynamic Huffman codes, allowing for a simpler decoder for special applications.